src/app/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import BookmarkIcon from '../assets/BookmarkIcon';
+import HomeIcon from '../assets/HomeIcon';
+import SearchIcon from '../assets/SearchIcon';
+import styles from './Navigation.module.css';
+
+function Navigation(): JSX.Element {
+  const [activeLink, setActiveLink] = useState<string>('home');
+  const active = {
+    fill: 'var(--color-primary)',
+    stroke: 'var(--color-primary)',
+  };
+
+  const inactive = {
+    fill: 'transparent',
```

Comment:
This might fix the filled svg problem in inactive state. We did this kind of similary in the movieButton component.

```suggestion
    fill: activeLink !== 'search' ? 'var(--text-primary)' : 'transparent',
```